packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/index.md

```diff
@@ -113,66 +114,119 @@ For password-only authentication, you need to update the password authenticator
 
 ### Set up your Okta org for a password-optional use case
 
-Set up your Okta org to enable authentication without a password using the followings steps:
+For password-optional authentication, you first need to
 
-1. [Set up the email authenticator](#set-up-the-email-authenticator).
-1. [Create a password-optional authenticator policy](#create-a-password-optional-authenticator-policy).
-1. [Add a new Global session policy](#add-a-new-global-session-policy).
-1. [Add a new authentication policy](#add-a-new-authentication-policy).
+1. [Set up the email authenticator for authentication and recovery](#set-up-the-email-authenticator).
+2. [Create a separate group for password-optional users](#create-a-group-for-password-optional-users).
 
-#### Set up the email authenticator
+To ensure that only specific app integrations can let users **sign up** without a password, do the following:
 
-1. Open the **Admin Console** for your org.
-1. Choose **Security** > **Authenticators** to show the available authenticators.
-1. On the **Setup** tab, locate the **Email** authenticator, and then select **Actions** > **Edit**.
-1. Set **This authenticator can be used for** to **Authentication and recovery**.
-1. Click **Save**.
+1. [Create a profile enrollment policy for password-optional users](#create-a-profile-enrollment-policy-for-password-optional-users) that adds them to the new group and assign your app to it.
```

Comment:
assign to "assigns"

Reply:
Assign is correct here. You must assign your app to the new policy.